uaclient/tests/test_system.py

```diff
@@ -97,6 +97,23 @@ class TestGetKernelInfo:
                     flavor="generic",
                 ),
             ),
+            (
+                "5.4.0-1021-aws-fips",
+                "Ubuntu 5.4.0-1021.21+fips2-aws-fips 5.4.44",
+                system.KernelInfo(
+                    uname_release="5.4.0-1021-aws-fips",
+                    proc_version_signature_full="Ubuntu 5.4.0-1021.21+fips2-aws-fips 5.4.44",  # noqa: E501
+                    proc_version_signature_version="5.4.0-1021.21+fips2-aws-fips",  # noqa: E501
+                    version="5.4.0",
+                    major=5,
+                    minor=4,
+                    patch=0,
+                    abi="1021",
+                    subrev="21",
+                    hwerev="",
+                    flavor="fips2-aws-fips",
```

Comment:
looking at this again @lucasmoura wouldn't we expect the flavor to just be `aws-fips`? I think `fips2` might be a different property of the kernel version that we need to add to our kernelinfo object